src/pages/resultsView/download/DownloadTab.tsx

```diff
@@ -277,10 +298,11 @@ export default class DownloadTab extends React.Component<IDownloadTabProps, {}>
                                 isLoading={false}
                                 className="pull-left forceHeaderStyle h4"
                             />
-                            {this.oqls.isComplete && this.caseAlterationData.isComplete && this.props.store.alterationsBySelectedMolecularProfiles.isComplete && (
+                            {this.oqls.isComplete && this.caseAlterationData.isComplete && this.props.store.alterationsBySelectedMolecularProfiles.isComplete && this.trackLabels.isComplete && (
```

Comment:
i think we know these are complete because of wrapping switch statement

Reply:
also, it should not be "default" case, because if it's default case, it's not complete!  but i know that is an earlier mistake, so we can leave it.

Reply:
@alisman I think you need default to be defined in order to get typescript to realize that you've exhausted all options. it will never actually go through because the other three cases exhaust the possible results.